discovery/eureka/eureka.go

```diff
@@ -35,12 +35,34 @@ const (
 	// in this discovery.
 	metaLabelPrefix = model.MetaLabelPrefix + "eureka_"
 
-	appNameLabel              = metaLabelPrefix + "app_name"
-	appInstanceHostNameLabel  = metaLabelPrefix + "app_instance_hostname"
-	appInstanceIPAddrLabel    = metaLabelPrefix + "app_instance_ipaddr"
-	appInstanceStatusLabel    = metaLabelPrefix + "app_instance_status"
-	appInstanceIDLabel        = metaLabelPrefix + "app_instance_id"
-	appInstanceMetadataPrefix = metaLabelPrefix + "app_instance_metadata_"
+	appNameLabel                                      = metaLabelPrefix + "app_name"
+	appInstanceHostNameLabel                          = metaLabelPrefix + "app_instance_hostname"
+	appInstanceHomePageURLLabel                       = metaLabelPrefix + "app_instance_homepageurl"
+	appInstanceStatusPageURLLabel                     = metaLabelPrefix + "app_instance_statuspageurl"
+	appInstanceHealthCheckURLLabel                    = metaLabelPrefix + "app_instance_healthcheckurl"
+	appInstanceIPAddrLabel                            = metaLabelPrefix + "app_instance_ipaddr"
+	appInstanceVipAddressLabel                        = metaLabelPrefix + "app_instance_vipaddress"
+	appInstanceSecureVipAddressLabel                  = metaLabelPrefix + "app_instance_securevipaddress"
+	appInstanceStatusLabel                            = metaLabelPrefix + "app_instance_status"
+	appInstancePortLabel                              = metaLabelPrefix + "app_instance_port"
+	appInstancePortEnabledLabel                       = metaLabelPrefix + "app_instance_port_enabled"
+	appInstanceSecurePortLabel                        = metaLabelPrefix + "app_instance_secureport"
+	appInstanceSecurePortEnabledLabel                 = metaLabelPrefix + "app_instance_secureport_enabled"
+	appInstanceDataCenterInfoNameLabel                = metaLabelPrefix + "app_instance_datacenterinfo_name"
+	appInstanceDataCenterInfoMetadataAmiLaunchIndex   = metaLabelPrefix + "app_instance_datacenterinfo_metadata_amilaunchindex"
+	appInstanceDataCenterInfoMetadataLocalHostname    = metaLabelPrefix + "app_instance_datacenterinfo_metadata_localhostname"
```

Comment:
```suggestion
	appInstanceDataCenterInfoMetadataLocalHostname    = metaLabelPrefix + "app_instance_datacenter_local_hostname"
```

Are the datacenter metadata a free map or the keys are fixed?

Reply:
The datacenter metadata's keys are fixed.